### ISSUE TITLE
Why GAN can converge and learn from limited labeled images well?

### ISSUE THREAD

**Issue by QiushiYang** [nv-tlabs/semanticGAN_code#10]
Thanks for sharing codes of your very interesting work. While I have a question about the work:
According to the Motivation-Sec. training GAN can avoid overfitting limited labeled training subset, but I wonder whether the GAN will also overfit or struggle from model collapse with only limited labeled training data (limited real images), or the GAN will be hard to converge?
Many thanks!

**Comment by lidaiqing** [comment]
@QiushiYang Thank you for your interest, we do assume the labeled training data is smaller comparing to unlabeled data. To avoid the generator overfit to the small set of labeled images, we have a stop-gradient operation to prevent gradient from Mask Discriminator to the image branch, which we found very useful.

**Comment by QiushiYang** [comment]
Oh, it sounds reasonable. I will try it. Thank a lot!
